.bazelci/presubmit.yml

```diff
@@ -53,6 +54,14 @@ tasks:
     - "--enable_runfiles" # this is not enabled by default on windows and is necessary for the cargo build scripts
     windows_targets: &windows_targets
     - "--" # Allows negative patterns; hack for https://github.com/bazelbuild/continuous-integration/pull/245
+    - "..."
+    - "-//bindgen/..."
+    - "-//test/load_arbitrary_tool/..."
+    - "-//test/test_env/..."
+    - "-//test/proto/..."
+    - "-//tools/rust_analyzer/..."
+    - "-//tools/runfiles/..."
```

Comment:
I think runfiles is meant to work on Windows? (Not sure about the others). But also, definitely let's merge and try to shrink this list over time :)

Reply:
Yup, I'll file issues for those.